src/modules/neo4j/README.md

```diff
@@ -0,0 +1,50 @@
+# neo4j
+
+Neo4j is a highly scalable, robust native graph database.
+Let's test it!
+
+## Examples
+
+Start a neo4j container and create a person node
+
+```typescript
+import neo4j from "neo4j-driver";
+import { Neo4jContainer } from "testcontainers";
+
+const container = await new Neo4jContainer().start();
+
+const driver = neo4j.driver(
+  "bolt://localhost:" + container.getMappedPort(Neo4jContainer.defaultBoltPort),
+  neo4j.auth.basic(Neo4jContainer.defaultUserName, Neo4jContainer.defaultPassword)
```

Comment:
instead of exposing constants, it'd be nice if the container itself could expose these, eg:
`container.getUsername()` and `container.getPassword()`
that way the constants can be made private and users of the container don't need to know where to find these constants

Reply:
I had it exactly like you suggested before and then changed it to the current implementation. Didn't make me happy either. 
It thought it is maybe confusing with container and startedContainer. My preferred way would be to have it on the startedContainer like getMappedPort, but I guess this more complicated because the base class returns a StartedContainer instance.  But for know I revert it back to my initial approach like you suggested to.